### ISSUE TITLE
NoSuchMethodError in AlertDialog usage

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/compose-mpp#1]
When it is used in [augene-ng use-compose-mpp-lib branch](https://github.com/atsushieno/augene-ng/actions/runs/1246438047) and AlertDialog is being created, it results in weird crash:

```
Exception in thread "main" java.lang.NoSuchMethodError: 'void dev.atsushieno.composempp.material.AlertDialogKt.AlertDialog-EN_aC5E(kotlin.jvm.functions.Function0, kotlin.jvm.functions.Function2, androidx.compose.ui.Modifier, kotlin.jvm.functions.Function2, kotlin.jvm.functions.Function2, kotlin.jvm.functions.Function2, androidx.compose.ui.graphics.Shape, long, long, androidx.compose.runtime.Composer, int, int)'
	at dev.atsushieno.augene.gui.AppKt$App$1.invoke(App.kt:95)
	at dev.atsushieno.augene.gui.AppKt$App$1.invoke(App.kt:31)
```

It looks like an issue in Compose Compiler plugin, but no further idea.

**Comment by atsushieno** [comment]
This seems related https://issuetracker.google.com/issues/194625542

**Comment by atsushieno** [comment]
fixed at 9e62c9a.
